Buildings/Controls/OBC/FDE/DOAS/EnergyWheel.mo

```diff
@@ -0,0 +1,217 @@
+          {60,28}}, color={255,0,255}));
+  connect(and3.y, erwStart)
+    annotation (Line(points={{84,20},{122,20}},
+                                              color={255,0,255}));
+  connect(conPID.u_s, erwT)
+    annotation (Line(points={{-92,-54},{-122,-54}}, color={0,0,127}));
+  connect(erwT, conPID1.u_s) annotation (Line(points={{-122,-54},{-96,-54},{-96,
+          -84},{-92,-84}}, color={0,0,127}));
+  connect(conPID1.u_m, supPrimSP) annotation (Line(points={{-80,-96},{-98,-96},{
+          -98,-86},{-122,-86}}, color={0,0,127}));
+  connect(supPrimSP, conPID.u_m) annotation (Line(points={{-122,-86},{-98,-86},{
+          -98,-66},{-80,-66}}, color={0,0,127}));
+  connect(conPID.y, max.u1) annotation (Line(points={{-68,-54},{-66,-54},{-66,-62},
+          {-64,-62}}, color={0,0,127}));
+  connect(conPID1.y, max.u2) annotation (Line(points={{-68,-84},{-66,-84},{-66,-74},
+          {-64,-74}}, color={0,0,127}));
+  connect(max.y, swi.u1)
+    annotation (Line(points={{-40,-68},{64,-68}}, color={0,0,127}));
+  connect(and3.y, swi.u2) annotation (Line(points={{84,20},{88,20},{88,-58},{56,
+          -58},{56,-76},{64,-76}}, color={255,0,255}));
+  connect(con0.y, swi.u3)
+    annotation (Line(points={{52,-84},{64,-84}}, color={0,0,127}));
+  connect(swi.y, erwSpeed)
+    annotation (Line(points={{88,-76},{122,-76}}, color={0,0,127}));
+  connect(ecoMode, and1.u2) annotation (Line(points={{-122,50},{-94,50},{-94,58},
+          {-66,58}}, color={255,0,255}));
+  connect(supFanProof, and1.u1) annotation (Line(points={{-122,80},{-94,80},{-94,
+          66},{-66,66}}, color={255,0,255}));
+  connect(and1.y, or2.u1)
+    annotation (Line(points={{-42,66},{60,66}}, color={255,0,255}));
+  connect(not2.y, or2.u2) annotation (Line(points={{56,50},{58,50},{58,58},{60,58}},
+        color={255,0,255}));
+  connect(and3.y, not2.u) annotation (Line(points={{84,20},{88,20},{88,36},{28,36},
+          {28,50},{32,50}}, color={255,0,255}));
+  connect(or2.y, bypDam)
+    annotation (Line(points={{84,66},{122,66}}, color={255,0,255}));
+  annotation (defaultComponentName="ERWcon",
+    Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+            Rectangle(extent={{-100,100},{100,-100}},
+            lineColor={0,0,0},
+            radius=10,
+            fillColor={255,255,255},
+            fillPattern=FillPattern.Solid),
+           Text(
+            extent={{-88,180},{92,76}},
+            lineColor={28,108,200},
+            textStyle={TextStyle.Bold},
+            textString="%name"),
+        Text(
+          extent={{-94,78},{-50,64}},
+          lineColor={28,108,200},
+          textString="supFanProof")}),
+    Diagram(coordinateSystem(preserveAspectRatio=false)),
+    Documentation(revisions="<html>
+<ul>
+<li>
+September 15, 2020, by Henry Nickels:</br>
+First implementation.</li>
+</ul>
+</html>", info="<html>
+<h4>Energy Recovery Wheel Start/Stop.</h4>
```

Comment:
If understand correctly, the ERW will be enabled when:
- the DOAS is energized (`supFanProof=true`), **and**
- the absolute difference between return air temperature (`rat`) and outside air temperature (`oat`) is greater than the recovery setpoint (`recSet`).

It should be disabled when:
- the DOAS is not energized, **or**,
- economizer mode is enabled (`ecoMode=true`), **or**,
- the RAT and OAT difference falls below the setpoint for longer than threshold time.

The implementation of enable/disable ERW will need to be corrected.